var/spack/repos/builtin/packages/gimp/package.py

```diff
@@ -0,0 +1,100 @@
+
+    maintainers = ["benkirk"]
+
+    conflicts("platform=darwin", msg="spack/GIMP currently requires Linux")
+    conflicts("platform=windows", msg="spack/GIMP currently requires Linux")
+
+    version("2.10.32", sha256="3f15c70554af5dcc1b46e6dc68f3d8f0a6cc9fe56b6d78ac08c0fd859ab89a25")
+    version("2.10.30", sha256="88815daa76ed7d4277eeb353358bafa116cd2fcd2c861d95b95135c1d52b67dc")
+    version("2.10.28", sha256="4f4dc22cff1ab5f026feaa2ab55e05775b3a11e198186b47bdab79cbfa078826")
+    version("2.10.26", sha256="5ddbccf1db462a41df9a26197fcb0d24c7152753a36b3c8b8a9506b4136395f7")
+    version("2.10.24", sha256="bd1bb762368c0dd3175cf05006812dd676949c3707e21f4e6857435cb435989e")
+
+    variant("doc", default=True, description="Build documentation with gtk-doc")
+    variant("ghostscript", default=True, description="Build with ghostscript support")
+    variant("jpegxl", default=True, description="Build with JPEG XL image format support")
+    # variant(
+    #     "libheif",
+    #     default=False,
+    #     description="Build with the libheif HEIF and AVIF file format decoder and encoder."
+    # )
+    variant(
+        "libmng", default=True, description="Build with Multiple-Image Network Graphics support"
+    )
+    variant(
+        "libwmf",
+        default=True,
+        description="Build with libwmf Windows Windows Metafile Format (WMF) support",
+    )
+    variant("libxpm", default=True, description="Build with libxpm support")
+    variant("webp", default=True, description="Build with WebP support")
+    # variant("python",      default=False, description="Build with Python bindings")
+
+    # ref. https://www.gimp.org/source/
+    depends_on("pkgconfig", type="build")
+    depends_on("babl")
+    depends_on("fontconfig@2.12.4:")
+    depends_on("gegl")
+    depends_on("gexiv2")
+    depends_on("ghostscript", when="+ghostscript")
+    depends_on("glib")
+    depends_on("glib-networking")
+    depends_on("gtk-doc", when="+doc")
+    depends_on("gtkplus@2.24.32:2.24.100")
+    depends_on("intltool")
+    depends_on("jpeg")
+    depends_on("libexif")
+    # depends_on("libheif+libde265", when="+libheif")
+    depends_on("libjxl", when="+jpegxl")
+    depends_on("libmng", when="+libmng")
+    depends_on("libmypaint@1.4")
+    depends_on("libpng")
+    depends_on("librsvg")
+    depends_on("libtiff")
+    depends_on("libwmf", when="+libwmf")
+    depends_on("libwebp+libwebpmux+libwebpdemux+libwebpdecoder+gif+jpeg+png+tiff", when="+webp")
+    depends_on("libxcursor")
+    depends_on("libxpm", when="+libxpm")
+    depends_on("mypaint-brushes@1.3")
+    depends_on("openexr")
+    depends_on("openjpeg")
+    # depends_on("python@3.6:", when="+python") # coming in 2.99
+    depends_on("pango@1.29.4:")
+    depends_on("poppler+glib")
+    depends_on("poppler-data@0.4.7:")
+    depends_on("zlib")
+
+    def configure_args(self):
+        args = [
+            "--prefix={0}".format(self.prefix),
```

Comment:
The base class already adds `--prefix` for you, so you can skip this

Reply:
Will do!